builtins/ja/facets/partials/instructions/loop-monitor-fix-replan-common.md

```
@@ -0,0 +1,7 @@
+**判断順序:**
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)